source/extensions/extensions_build_config.bzl

```diff
@@ -77,6 +77,7 @@ EXTENSIONS = {
     "envoy.stat_sinks.dog_statsd":                      "//source/extensions/stat_sinks/dog_statsd:config",
     "envoy.stat_sinks.metrics_service":                 "//source/extensions/stat_sinks/metrics_service:config",
     "envoy.stat_sinks.statsd":                          "//source/extensions/stat_sinks/statsd:config",
+    "envoy.stat_sinks.hystrix":                         "//source/extensions/stat_sinks/hystrix:config",
```

Comment:
@trabetti Small comment - the hystrix entry is the only one in the list that is not alphabetically sorted within the section. For consistency it should be below `dog_statsd` and above `metrics_service`

Reply:
opened #3752